rca/home/models.py

```diff
@@ -12,18 +15,71 @@ class HomePage(BasePage):
     # Only allow creating HomePages at the root level
     parent_page_types = ["wagtailcore.Page"]
 
-    strapline = models.CharField(blank=True, max_length=255)
-    call_to_action = models.ForeignKey(
-        "utils.CallToActionSnippet",
-        blank=True,
+    hero_image = models.ForeignKey(
+        get_image_model_string(),
         null=True,
+        blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    hero_colour_option = models.CharField(
+        max_length=1,
+        choices=(
+            ("1", "Light text on a dark image"),
```

Comment:
If we are going to use numbers, why not just use `PositiveSmallIntegerField`?

Reply:
I attempted this but it resulted in `django.db.utils.DataError: invalid input syntax for integer: ""`

Reply:
Did you change `"1"` to `1`?